drivers/platform/linux/linux_uart.c

```diff
@@ -105,7 +105,7 @@ int32_t uart_init(struct uart_desc **desc, struct uart_init_param *param)
 	linux_init = param->extra;
 
 	ret = snprintf(path, sizeof(path), "/dev/%s", linux_init->device_id);
-	if (ret < 0 || ret >= sizeof(path)) {
+	if (ret < 0 || ret >= (int)sizeof(path)) {
```

Comment:
let's try to use explicit size for types everywhere (if possible)
here, it's possible, you know what type ret is

Reply:
I don't really understand what do you want me to do :)